### DISCUSSION TITLE
Broken Hosted Checkout URL

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#276] [Help] [answered]
Hello,
I'm trying to use the Hosted Checkout feature by opening the URL provided as checkoutUrl on the cart object. When opened, the URL redirects to https://<my-store>.swell.store/checkout/null and displays a 404 page. What's going on? Also I should mention that it worked the first time.

**Comment by ericingram** [comment, marked as answer]
It could be an expired cart although it certaintly shouldn't redirect to `null`. You can adjust cart expiration time in Settings > Checkout.

We have some improvements in this area that will be deployed likely Tuesday, that may fix the issue you're seeing.

**Reply by igoraugustynski** [reply]
It's not expired, I tried recreating it multiple times. Can You provide some more support on this? I can provide You the cartId.

Edit: It was an expired cart indeed! For some reason the expiration time in Settings > Checkout changed to 0 minutes. Weird.
